short_answer/static/js/src/short_answer.js

```diff
@@ -185,8 +206,26 @@ function ShortAnswerXBlock(runtime, element, options) {
     });
   });
 
+  /**
+   * Enable/Disable the submit button.
+   */
+  $shortAnswer.keyup(function() {
+    if (isSubmitAllowed()) {
+      $answerSubmitButton.removeAttr('disabled');
+    }
+    else {
+      $answerSubmitButton.attr('disabled', true);
+    }
+  });
+
   $(function($) {
     const csvDownloadUrl = runtime.handlerUrl(element, 'csv_download');
     $('form[name=csv-download]', element).attr('action', csvDownloadUrl);
+
+    $shortAnswer.keyup();
```

Comment:
What's the meaning of this line?

Reply:
When the page is first loaded this will disable (or not) the submit button. Since the function is reduced to one line I'll just replace this with that line for readability.